From 6369e5efe74f55bc62d215f5756775514ea341dd Mon Sep 17 00:00:00 2001
From: Julian Scheel <julian@jusst.de>
Date: Fri, 5 Oct 2012 12:46:47 +0200
Subject: [PATCH] Disable Werror for deprecated in teletextdec plugin.

The plugin uses deprecated functions in glib. As we will switch to a more
recent gstreamer version soon, just disable the fatal warning and keep using
the deprecated functions for now.

Signed-off-by: Julian Scheel <julian@jusst.de>
---
 ext/teletextdec/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/ext/teletextdec/Makefile.am b/ext/teletextdec/Makefile.am
index c153c56..6e063aa 100644
--- a/ext/teletextdec/Makefile.am
+++ b/ext/teletextdec/Makefile.am
@@ -2,7 +2,9 @@ plugin_LTLIBRARIES = libgstteletextdec.la
 
 libgstteletextdec_la_SOURCES = gstteletextdec.c teletext.c
 
-libgstteletextdec_la_CFLAGS = $(GST_PLUGINS_BASE_CFLAGS) $(GST_CFLAGS) $(TELETEXTDEC_CFLAGS)
+libgstteletextdec_la_CFLAGS = $(GST_PLUGINS_BASE_CFLAGS) $(GST_CFLAGS) \
+	$(TELETEXTDEC_CFLAGS) -Wno-deprecated-declarations
+
 libgstteletextdec_la_LIBADD = $(GST_PLUGINS_BASE_CFLAGS) $(GST_BASE_LIBS) $(TELETEXTDEC_LIBS)
 libgstteletextdec_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS)
 libgstteletextdec_la_LIBTOOLFLAGS = --tag=disable-static
-- 
1.7.12.2

